### PR TITLE
Module factory wrapper

### PR DESCRIPTION
**Pull request by victorwiebe**

### CODE REVIEW

File: src/contract_wrappers/modules/module_factory_wrapper.ts [hunk 1]
```diff
@@ -92,16 +140,174 @@ export default class ModuleFactoryWrapper extends ContractWrapper {
...
+   */
+  public changeSTVersionBounds = async (params: ChangeSTVersionBoundsParams) => {
+    await this.checkOnlyOwner(params.txData);
+    assert.assert(
+      params.boundType === BoundType.LowerBound || params.boundType === BoundType.UpperBound,
+      'Invalid bound type',
+    );
+    assert.assert(params.newVersion.length === 3, 'Invalid version, number array must have 3 elements');
+    const currentBound = BoundType.LowerBound === params.boundType ? await this.getLowerSTVersionBounds() : await this.getUpperSTVersionBounds();
+    for (let i = 0; i < 3; i += 1) {
```
[the remark this answers is not in the document]
Comment (victorwiebe) [reply src/contract_wrappers/modules/module_factory_wrapper.ts]: (1) Do I need @types/semver ?

File: src/contract_wrappers/modules/module_factory_wrapper.ts [hunk 2]
```diff
@@ -14,6 +14,7 @@ import {
 } from '@polymathnetwork/abi-wrappers';
 import { schemas } from '@0x/json-schemas';
 import assert from '../../utils/assert';
+import * as semver from 'semver';
```
Comment (monitz87) [inline src/contract_wrappers/modules/module_factory_wrapper.ts]: This can be written as `import semver from  'semver';`

Comment (victorwiebe) [reply src/contract_wrappers/modules/module_factory_wrapper.ts]: Issue with that is
"Module ... polymath-contract-wrappers/node_modules/@types/semver/index"' has no default export"

How do I resolve that?

Comment (monitz87) [reply src/contract_wrappers/modules/module_factory_wrapper.ts]: `yarn add --dev @types/semver` and add `"esModuleInterop": true,` to the compilerOptions in tsconfig.json

File: src/utils/filter_utils.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import * as ethUtil from 'ethereumjs-util';
 import * as jsSHA3 from 'js-sha3';
 import * as _ from 'lodash';
-import * as uuid from 'uuid/v4';
+import uuid from 'uuid/v4';
```
Comment (monitz87) [inline src/utils/filter_utils.ts]: Would you mind doing the same with lodash and the other packages that are being imported with * accross the project?

**Pull request merged**